models/ShipFrame.json

```diff
@@ -56,6 +59,8 @@
   "required": [
     "symbol",
     "name",
+    "condition",
+    "performance",
```

Comment:
perhaps this was meant to be renamed to integrity

Reply:
Good catch! Just pushed the fix.